packit_service_fedmsg/callbacks.py

```diff
@@ -0,0 +1,177 @@
+        what = f"[Koji] id:{event.get('id')} {event.get('old')}->{event.get('new')}"
+
+    return CallbackResult(msg=what)
+
+
+def _fedora_dg_push(topic: str, event: dict, packit_user: str) -> CallbackResult:
+    if getenv("PROJECT", "").startswith("packit") and not specfile_changed(
+        event,
+    ):
+        return CallbackResult(
+            msg="[Fedora DG] No specfile change, ignoring the push.",
+            pass_to_service=False,
+        )
+
+    if commit := event.get("commit"):
+        what = f"{commit.get('repo')} {commit.get('rev')}@{commit.get('branch')}"
+    else:
+        what = "Couldn't get commit out of the event."
+
+    return CallbackResult(msg=f"[Fedora DG] Passing push: {what}")
+
+
+def _fedora_dg_pr_flag(topic: str, event: dict, packit_user: str) -> CallbackResult:
+    if nested_get(event, "pullrequest", "user", "name") != packit_user:
+        return CallbackResult(
+            msg=f"[Fedora DG] Flag changed in a PR not created by {packit_user}, ignoring.",
+            pass_to_service=False,
+        )
+
+    return CallbackResult(
+        msg=(
+            "[Fedora DG] Flag on "
+            f"{nested_get(event, 'pullrequest', 'project', 'fullname')}"
+            f" changed to '{nested_get(event, 'flag', 'comment')}'"
+        ),
+    )
+
+
+def _fedora_dg_pr_comment(topic: str, event: dict, packit_user: str) -> CallbackResult:
+    project = nested_get(event, "pullrequest", "project", "fullname")
+    comments = nested_get(event, "pullrequest", "comments")
+    last_comment = comments[-1]
+    return CallbackResult(
+        msg=(
+            f"[Fedora DG] For {project}"
+            f" new comment: '{last_comment['comment']}'"
+            f" from {last_comment['user']['name']}"
+        ),
+    )
+
+
+def _fedora_dg_pr_closed(topic: str, event: dict, packit_user: str) -> CallbackResult:
+    project = nested_get(event, "pullrequest", "project", "fullname")
+
+    if not nested_get(event, "pullrequest", "merged"):
+        return CallbackResult(
+            msg=f"[Fedora DG] Pull request  in {project} was closed, ignoring.",
+            pass_to_service=False,
+        )
+
+    return CallbackResult(
+        msg=f"[Fedora DG] Merged pull request in {project}.",
+    )
+
+
+def _hotness_bugzilla(topic: str, event: dict, packit_user: str) -> CallbackResult:
+    package = event.get("package")
+    version = nested_get(event, "trigger", "msg", "project", "version")
+
+    return CallbackResult(
+        msg=f"[Hotness] New update of package {package} to version {version}.",
+    )
+
+
+def _anitya_version_update(topic: str, event: dict, packit_user: str) -> CallbackResult:
+    package = None
+    for p in nested_get(event, "message", "packages"):
+        if p.get("distro") == "CentOS":
+            package = p.get("package_name")
+            break
+
+    if package is None:
+        return CallbackResult(
+            msg=f"[Anitya] CentOS mapping is not configured for {package}, ignoring.",
```

Comment:
```suggestion
            msg=f"[Anitya] CentOS mapping is not configured for {nested_get(event, "project", "name")}, ignoring.",
```

the `package` is None here, isn't it?

Reply:
Yup… I'm a _pepege_